src/components/TodoInput/index.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+
+const ToDoInput = ({ addTodo }) => {
+  const [inputValue, setInputValue] = useState('');
+  const onClick = (text) => {
+    addTodo(text);
+    setInputValue('');
+  };
+  return (
```

Comment:
```suggestion
  const [inputValue, setInputValue] = useState('');

  const onClick = (text) => {
    addTodo(text);
    setInputValue('');
  };

  return (
```

서로 연관이 많은 것끼리 적절히 묶어주면 더 읽기 좋습니다

Reply:
단순히 코드 길이를 줄이려고 공백줄을 습관적으로 없에고 있었는데, 지양해야겠네요!